docker/README.md

```diff
@@ -1,10 +1,11 @@
 Docker
 ======
 
-You can run osmose-backend in a Docker container. The advantage is that
-you do not need to setup and configure Python, Java and PostgreSQL on
-your system. The PostgreSQL and eventually the Osmose Frontend are
-dependencies.
+osmose-backend can be run in a Docker container. This avoids setting
+up and configuring Python, Java and PostgreSQL on your system.
+The osmose-backend docker image requires a running instance of PostgreSQL
+(PostGIS). An instance of osmose-frontend is also required to display the
+issues on a web interface.
```

Comment:
PostgreSQL is not a requirement as it is integrated into the docker-compose setup.

I prefer avoid to encourage to use the osmose-frontend as the installation and the usage is more tedious than the backend.

Reply:
OK. My firt reason to suggest a change for this sentence is that "eventually" is a "faux ami" meaning "finalement / au final" and I think you may have meant "éventuellement"
Alternative proposals ;) :
- remove this last sentence completely or
- "Note : A PostgreSQL docker is automatically installed and run by docker-compose and doesn't need be installed manually." (no mention of osmose-frontend)
- "Note : A PostgreSQL docker is automatically installed and run by docker-compose and doesn't need be installed manually. The osmose-frontend docker may also be run but is not mandatory"

Reply:
The last.